test/integration/ads_integration_test.cc

```diff
@@ -236,6 +236,93 @@ TEST_P(AdsIntegrationTest, ClusterInitializationUpdateOneOfThe2Warming) {
   test_server_->waitForGaugeGe("cluster_manager.active_clusters", 4);
 }
 
+TEST_P(AdsIntegrationTest, DeltaSdsRemovals) {
```

Comment:
This test LGTM, but I wonder whether it should be here (ads_integration_test.cc) or in the sds integration tests.
Looking at the test it seems it only needs the sds to be updated, and if that's the case, I think it should be moved there.

Also, please add a short comment describing what the test validates and why it only relates to delta-xDS.

Reply:
Ah, I see.
LGTM, thanks!